workflows/train/deepspeed/cifar/job.py

```diff
@@ -36,27 +36,8 @@
     True,
 ]
 
-# create an environment
-# Note: We will use the Dockerfile method to create an environment for DeepSpeed.
-# In future, we plan to create a Curated environment for DeepSpeed.
-env = Environment(name="deepspeed-example")
-env.docker.enabled = True
-
-# indicate how to run Python
-env.python.user_managed_dependencies = True
-env.python.interpreter_path = "/opt/miniconda/bin/python"
-
-# To install any Python packages you need, simply add RUN pip install package-name to the docker string. E.g. `RUN pip install sklearn`
-# Specify docker steps as a string and use the base DeepSpeed Docker image
-dockerfile = r"""
-FROM deepspeed/base-aml:with-pt-ds-and-deps
-RUN pip install azureml-mlflow
-RUN echo "Welcome to the DeepSpeed custom environment!"
-"""
-
-# set base image to None, because the image is defined by dockerfile.
-env.docker.base_image = None
-env.docker.base_dockerfile = dockerfile
+# Use the DeepSpeed Curated Environment
+env = Environment.get(ws, name="AzureML-DeepSpeed-0.3-GPU")
```

Comment:
who maintains this? @saachigopal

Reply:
azuremlcli repo shows it was added yesterday by Ammar

Reply:
@awan-10 will you maintain this? is there any contact on the AML side helping? I do worry this will break over time

Reply:
@lostmygithubaccount - We have a list of maintainers including myself, Jeff Rasley, and the DeepSpeed team. We have talked to Vitaliy and Alex about it.